_posts/2020-12-15-insight-to-snaas-delivery-process.md

```diff
@@ -0,0 +1,57 @@
+---
+
+title: "Insight to SNaaS delivery process"
+author: [herflis, VargaJoe]
+image: "../img/posts/delivery.jpg"
+tags: [deployment, countinuous integration, saas, infra]
+redirect_to: https://www.sensenet.com/blog/2020-12-15-insight-to-snaas-delivery-process
+
+---
+
+We constantly working on new features, fixes and other awesomeness. Let’s see how these new things get to the customers.
+
+---
+
+At sensenet, we are working on making the product and our services better with fixes, improvements and new features everyday. Since we wanted to provide a predictable and reliable service in addition to being constantly updated and improved we had to work out a process to make the updates fairly frequent and seamless yet valuable. Following article details our delivery system.
```

Comment:
```suggestion
At sensenet, we are working on making the product and our services better with fixes, improvements and new features every day. Since we wanted to provide a predictable and reliable service in addition to being constantly updated and improved we had to work out a process to make the updates fairly frequent and seamless yet valuable. The following article details our delivery system.
```